BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift

```diff
@@ -193,6 +193,11 @@ public final class GiniBankConfiguration: NSObject {
       * Set an adapter implementation to show a custom illustration on the "QR code" onboarding page.
       */
     public var onboardingQRCodeIllustrationAdapter: OnboardingIllustrationAdapter?
+
+    /**
+     * Set an adapter implementation to show a custom illustration on the return assistant onboarding page.
+     */
+   public var returnAssistantOnboardingIllustrationAdapter: OnboardingIllustrationAdapter?
```

Comment:
[Request] To keep consistent with naming we should call this one `digitalInvoiceOnboardingIllustrationAdapter`.